docs/contributors/grammar.md

```diff
@@ -1,5 +1,5 @@
 
-# The Gutenberg block grammar
+# Block Grammar
```

Comment:
For some reason I thought this file was generated automatically, which might mean this change gets overridden. Anyone have context (cc @dmsnell @gziolo et al)?

Reply:
I think @mcsf should know best.

Reply:
Correct! Grammar generator introduced in #6116.

That particular heading should be changed in the generator itself:

https://github.com/WordPress/gutenberg/blob/0f5891556e8e3ec3e41fcc318533bba87983338b/bin/generate-public-grammar.js#L94-L99

Reply:
Good question, thanks I'll update there 👍